tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Settings/PartialViews.spec.ts

```diff
@@ -0,0 +1,235 @@
+﻿import {AliasHelper, ConstantHelper, test} from '@umbraco/playwright-testhelpers';
+import {expect} from "@playwright/test";
+
+test.describe('Partial Views tests', () => {
+  const partialViewName = 'TestPartialView';
+  const partialViewFileName = partialViewName + ".cshtml";
+  
+  test.beforeEach(async ({page, umbracoApi, umbracoUi}) => {
+    await page.goto(umbracoApi.baseUrl + '/umbraco');
+    await umbracoUi.goToSection(ConstantHelper.sections.settings);
+  });
+
+  async function gotoPartialViewDetail(page, partialViewFileName) {
+    await page.locator('umb-tree-item').filter({ hasText: 'Partial Views' }).locator('#caret-button').click();
+    await page.locator('umb-tree-item').getByLabel(partialViewFileName).click();
+  } 
+
+  test('can create an empty partial view', async ({page, umbracoApi}) => {
+    //Arrange
+    await umbracoApi.partialView.ensureNameNotExistsAtRoot(partialViewName);
+
+    // Act
+    await page.locator('umb-tree-item', {hasText: 'Partial Views'}).getByLabel('Open actions menu').click({force: true});
+    await page.getByLabel('New empty partial view').click();
+    await page.getByLabel('template name').fill(partialViewName);
+    await page.waitForLoadState('networkidle');
```

Comment:
I'm a bit unsure if we want to use the 'networkidle'. Playwright themselves discourage it https://playwright.dev/docs/api/class-page#page-wait-for-load-state

Reply:
ok, I will avoid to using it